spec/model/parse_spec.rb

```diff
@@ -342,4 +342,30 @@ def to_params
       expect(user.to_params).to eql(user: { first_name: "Someone" })
     end
   end
+
+  context 'when passed a non-Her ActiveModel instance' do
+    before do
+      klass = Class.new do
+        include ActiveModel::Serialization
+
+        def attributes
+          { name: nil }
+        end
+
+        def name
+          'foo'
+        end
+      end
+
+      @model = klass.new
+
+      Her::API.setup
+      spawn_model 'Foo::User'
+    end
+
+    it 'serializes the instance in `to_params`' do
+      attributes = { model: @model }
+      expect(Foo::User.to_params(attributes)).to eq(model: { name: 'foo' })
```

Comment:
Let's test the public version of `#to_params` since `.to_params` is marked private.

```ruby
    it 'serializes the instance in `to_params`' do
      attributes = { model: @model }
      user = Foo::User.new(attributes)
      expect(user.to_params).to eq(model: { name: 'foo' })
    end
```